app/views/subscriptions/check_email.html.erb

```diff
@@ -1,3 +1,9 @@
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+      href: :back
```

Comment:
Should this be `@back_url`?

Reply:
updated that and added a new ```@back_url``` definition